actors/evm/src/interpreter/address.rs

```diff
@@ -79,6 +82,10 @@ impl EthAddress {
         Some(u64::from_be_bytes(self.0[12..].try_into().unwrap()))
     }
 
+    pub fn is_precompile<RT: Runtime>(&self) -> bool {
```

Comment:
This is a bit funky to have tied to the `EthAddress` type.

Reply:
Given that we call this _once_, I wouldn't define this.

Reply:
Note: "is precompile" isn't a property of the address, it's "is this a precompile in this specific runtime". That's why this is a bit funky.